websites/A/Animes House/presence.ts

```diff
@@ -0,0 +1,108 @@
+var presence = new Presence ({
+    clientId: "711685584573169686"
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
+
+function getTimestamps(videoTime, videoDuration): any {
+    var startTime = Date.now();
+    var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+    return [Math.floor(startTime / 1000), endTime]; 
+}

```

Comment:
Don't do `: any` use  `: Array<number>`
var starttime can be with const, same with end time.